DataCapture/Utils.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+
```

Comment:
Can this move to a common header file outside of plugins so that it can be used by all?